onnxconverter_common/float16.py

```diff
@@ -6,7 +6,7 @@
 import itertools
 import numpy as np
 import onnx
-from distutils.version import StrictVersion
+from verlib import NormalizedVersion as V
```

Comment:
I looked into verlib, it seems that the package was not updated since 2012. I think it would be better either to do it manually or to rely on something more recent and maintained.

Reply:
@xadupre Got your point. how about we use distuils.version.LooseVersion?

Reply:
Everything from python distribution is ok.

Reply:
@xadupre pls review again :)